packages/driver/src/cypress/runner.ts

```diff
@@ -74,10 +74,10 @@ const testBeforeRunAsync = (test, Cypress) => {
   })
 }
 
-const testBeforeAfterRunAsync = (test, Cypress) => {
+const testBeforeAfterRunAsync = (test, Cypress, additionalInfo) => {
```

Comment:
`additionalInfo` is a pretty lackluster argument name.

If you don't want to name the arg, you could do something like this i think.
```js
const testBeforeAfterRunAsync = (...args) => {
  const {test} = args
  return Promise.try(() => {
    if (!fired(TEST_BEFORE_AFTER_RUN_ASYNC_EVENT, test)) {
      return fire(TEST_BEFORE_AFTER_RUN_ASYNC_EVENT, ...args)
    }
  })
```

Reply:
Good call: [`ee5277a` (#27522)](https://github.com/cypress-io/cypress/pull/27522/commits/ee5277a17d303bad49e01a4addc8c0924e87735b)